compute/instances.go

```diff
@@ -395,14 +406,26 @@ func (c *InstancesClient) Rename(ctx context.Context, input *RenameInstanceInput
 type ReplaceTagsInput struct {
 	ID   string
 	Tags map[string]string
+	CNS  InstanceCNS
+}
+
+// toAPI() merges both Tags and CNS tags into the same map of strings used as
+// the body of our ReplaceTags request.
+func (input ReplaceTagsInput) toAPI() map[string]interface{} {
+	result := map[string]interface{}{}
+	for key, value := range input.Tags {
+		result[key] = value
```

Comment:
I think we're alright since we're liberal with what can be a tag and we're also relying on the JSON encoder to marshal types going into CloudAPI. But I can see how the use of `interface{}` would normally allow other types through in cases like this.